cmd/lukso/init.go

```diff
@@ -1,11 +1,12 @@
 package main
 
 import (
+	"os"
+	"strings"
+
 	"github.com/m8b-dev/lukso-cli/config"
 	"github.com/m8b-dev/lukso-cli/pid"
 	"github.com/urfave/cli/v2"
```

Comment:
is this format done by linter? If so, which one are you using? The way that I know of sorting imports looks something like this:

```go
import (
    // Section for builtins
    "builtin/package1"
    "builtin/package1"
    // Section for external dependencies
    "github.com/repo/external-library"
    "github.com/other-repo/other-external-library"

    // Section for internal packages
    "github.com/lukso-network/tools-lukso-cli/some-internal-package"
    "github.com/lukso-network/tools-lukso-cli/some-other-internal-package"
)
```
And I know that some linters are configurable, with customizable local module prefix (f.e. for sorting purposes)

But it is totally fine, those are just imports, so it doesn't matter that much

Reply:
I think it comes from this one:

https://golangci-lint.run/usage/linters/#goimports

`goimports`

I'm using VSCode and I have format on save enabled, it reorders auto. Maybe this comes from `gofmt`